tests/integration/io-threads.tcl

```diff
@@ -0,0 +1,90 @@
+proc get_io_threads_active_num {client} {
+    set server_info [$client info server]
+    regexp {io_threads_active_num:([0-9]+)} $server_info ->\
+        io_threads_active_num
+    return $io_threads_active_num
+}
+
+proc start_benchmark {thread_num client_num} {
+    set bench_pid [
+        exec src/valkey-benchmark -s [srv unixsocket] -t set \
+        -r 5000000 -n 10000000 --threads $thread_num -c $client_num \
+        > /dev/null &
+    ]
+    return $bench_pid
+}
+
+tags {"io-threads external:skip"} {
+    test "Disable io-threads" {
+        start_server {overrides {"io-threads" {1} "io-threads-do-reads" {no}}} {
+            set master_pid [srv 0 pid]
+            set pstree_out [exec ps -T -p $master_pid]
+            assert_no_match "*io_thd_*" $pstree_out
+            assert_match "*io_threads_active:0*" [r info server]
+            assert_match "*io_threads_maximum_num:1*" [r info server]
+            assert_match "*io_threads_active_num:1*" [r info server]
+
+            assert_equal {OK} [r set k v]
+            assert_equal {1} [r DBSIZE]
+        }
+    }
+
+    test "Enable io-threads" {
+        start_server {overrides {"io-threads" {4} "io-threads-do-reads" {yes}}} {
+            set master_pid [srv 0 pid]
+            set pstree_out [exec ps -T -p $master_pid]
+            assert_match "*io_thd_1*" $pstree_out
+            assert_match "*io_thd_2*" $pstree_out
+            assert_match "*io_thd_3*" $pstree_out
+            set server_info [r info server]
+            regexp {io_threads_active:([0-9]+)} $server_info -> \
+                io_threads_active
+            regexp {io_threads_maximum_num:([0-9]+)} $server_info -> \
+                io_threads_maximum_num
+            regexp {io_threads_active_num:([0-9]+)} $server_info -> \
+                io_threads_active_num
```

Comment:
```suggestion
            set io_threads_active [get_info_field $server_info io_threads_active]
            set io_threads_maximum_num [get_info_field $server_info io_threads_maximum_num]
            set io_threads_active_num [get_info_field $server_info io_threads_active_num]
```

Reply:
Thanks, `proc get_info_field` is a common proc in cluster module, I just moved it to `support/util.tcl`.